scripts/getPullRequestsMergedBetweenTest.sh

```diff
@@ -3,11 +3,12 @@
 # Fail immediately if there is an error thrown
 set -e
 
-TEST_DIR=$(dirname "$(dirname "$(cd "$(dirname "$0")" || exit 1;pwd)/$(basename "$0")")")
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Working locally and on CI: https://github.com/Expensify/App/runs/5281327483?check_suite_focus=true#step:8:1